pages/app/models/refinery/page.rb

```diff
@@ -312,11 +312,11 @@ def content_for?(part_title)
     #    ::Refinery::Page.first.part_with_title(:body)
     #
     # Will return the Refinery::PagePart object with that title using the first page.
-    def part_with_title(part_title)
+    def part_with_slug(part_slug)
```

Comment:
Can you please deprecate the old API, and update the documentation above this method to be correct?
Same thing needs to happen for `content_for`,  and `content_for?`